### ISSUE TITLE
load/read/write files ubyte|ubyte.gz

### ISSUE THREAD

**Issue by SalAlba** [SalAlba/tips-tricks-pyhton#1]
How can we read data ubyte|ubyte.gz

The MNIST collection is available at the following links:

(training set):
http://yann.lecun.com/exdb/mnist/train-images-idx3-ubyte.gz
http://yann.lecun.com/exdb/mnist/train-labels-idx1-ubyte.gz

(validation set):
http://yann.lecun.com/exdb/mnist/t10k-images-idx3-ubyte.gz
http://yann.lecun.com/exdb/mnist/t10k-labels-idx1-ubyte.gz

**Comment by SalAlba** [comment]
Extract images from .idx3-ubyte file or GZIP via Python
